FirebaseAdmin/FirebaseAdmin/Messaging/WebpushFcmOptions.cs

```diff
@@ -28,5 +28,24 @@ public sealed class WebpushFcmOptions
         /// </summary>
         [JsonProperty("link")]
         public string Link { get; set; }
+
+        /// <summary>
+        /// Copies this Webpush FCM options, and validates the content of it to ensure that it can
+        /// be serialized into the JSON format expected by the FCM service.
+        /// </summary>
+        internal WebpushFcmOptions CopyAndValidate()
+        {
+            if (this.Link != null && !this.Link.StartsWith("https"))
```

Comment:
Can we also check the well-formedness of the URL by running `Uri.IsWellFormedUriString(copy.Link, UriKind.Absolute)`?